sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/jms/ServiceBusJmsAutoConfigurationTest.java

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.jms;
+
+import com.azure.spring.cloud.autoconfigure.jms.properties.AzureServiceBusJmsProperties;
+import org.apache.qpid.jms.JmsConnectionFactory;
+import org.junit.jupiter.api.Test;
+import org.messaginghub.pooled.jms.JmsPoolConnectionFactory;
+import org.springframework.boot.autoconfigure.AutoConfigurations;
+import org.springframework.boot.test.context.FilteredClassLoader;
+import org.springframework.jms.connection.CachingConnectionFactory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ServiceBusJmsAutoConfigurationTest extends AbstractServiceBusJmsAutoConfigurationTest {
+    ServiceBusJmsAutoConfigurationTest() {
+        this.contextRunner = super.contextRunner
+            .withPropertyValues("spring.jms.servicebus.pricing-tier=premium")
+            .withConfiguration(AutoConfigurations.of(ServiceBusJmsAutoConfiguration.class));
+    }
+
+    @Test
+    void testAzureServiceBusAutoConfiguration() {
```

Comment:
we can omit the prefix `test` for all test method names.

Reply:
ok